static/js/egi.js

```diff
@@ -0,0 +1,288 @@
+jQuery(document).ready(function () {
```

Comment:
I saw that you disabled the check of this file, but you didn't fixed any of the problems like at least the inconsistent indentation, it would already have been better to have some default rules enforced instead of nothing at all.
Please also add an header/comments to clarify why we have this/the purpose

Reply:
I disagree with some of the rules being enforced (e.g. no ; at the end of statements). 
I did an auto-format on it, but did not like the results. I may have to work on the rules for JavaScript. 

But the biggest problem is that it seems that this is being checked as it was an independent file, although this gets injected by some Go code into a HTML fragment (the page's head) after e.g. jQuery was already included. The linter flags a massive amount of errors on anything that starts with $.

Reply:
Added header